packages/core/workers/src/Profiler.js

```diff
@@ -39,17 +39,23 @@ type PositionTickInfo = {|
 export default class Profiler {
   session: Session;
 
-  startProfiling() {
-    this.session = new Session();
-    this.session.connect();
+  async startProfiling() {
+    try {
+      const inspector: any = await require('inspector');
+      this.session = new inspector.Session();
+      this.session.connect();
 
-    return Promise.all([
-      this.sendCommand('Profiler.setSamplingInterval', {
-        interval: 100
-      }),
-      this.sendCommand('Profiler.enable'),
-      this.sendCommand('Profiler.start')
-    ]);
+      return Promise.all([
+        this.sendCommand('Profiler.setSamplingInterval', {
+          interval: 100
+        }),
+        this.sendCommand('Profiler.enable'),
+        this.sendCommand('Profiler.start')
+      ]);
+    } catch (err) {
+      /* eslint-disable no-console */
+      console.error('Profiling not available', err);
```

Comment:
This way, in `sendCommand` `invariant(this.session != null);` would throw an unhelpful error. So we should throw an error here instead of `console.error`.

Reply:
Well, `try {...} catch(e){ throw e; }` is redundant. The error should actually tell the user that the Parcel flag `--profile` flag is causing this and errors should only be caught for `require('inspector')`, not for the commands after that
Also, the `any` in `const inspector: any = ...` isn't needed